kafka-rest/src/main/java/io/confluent/kafkarest/resources/v3/ProducerMetrics.java

```diff
@@ -67,6 +68,10 @@ final class ProducerMetrics {
   static final String REQUEST_SIZE_AVG_METRIC_NAME = "request-size-avg";
   private static final String REQUEST_SIZE_AVG_METRIC_DOC = "The average request size in bytes.";
 
+  static final String REQUEST_SIZE_WINDOWED_SUM_METRIC_NAME = "request-size-windowed-sum";
+  private static final String REQUEST_SIZE_WINDOWED_SUM_METRIC_DOC =
+      "The windowed sum of request sizes in bytes.";
+
   static final String REQUEST_COUNT_WINDOWED_METRIC_NAME = "request-count-windowed";
   private static final String REQUEST_COUNT_WINDOWED_METRIC_DOC =
       "The total number of requests sent within the given window.";
```

Comment:
nit: I think a doc that's very close to the windowed request count below is a better fit - e.g. "The summed size in bytes of requests sent within the given window".